app/database/schema/users.schema.ts

```
@@ -1,5 +1,7 @@
 import { relations, sql } from "drizzle-orm";
 import { boolean, index, integer, pgEnum, pgTable, serial, text, timestamp, uuid } from "drizzle-orm/pg-core";
+import { createInsertSchema } from "drizzle-zod";
```

ESLint

[error] Unable to resolve path to module 'drizzle-zod'.

(import-x/no-unresolved)